client/flutter/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   page_view_indicator: ^2.0.4
   flutter_staggered_grid_view: ^0.3.0
   flutter_html: ^0.11.1
+  firebase_analytics: ^5.0.11
```

Comment:
We need to add firebase_core as a dependency -- see https://firebase.google.com/docs/flutter/setup

Reply:
It's not actually required for analytics afaik. Will ask in slack.